packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js

```diff
@@ -27,6 +27,7 @@ your environment (operating system, application framework, etc.).
 `;
 
 const CreateIssueButton = ({
+  labels = ['feedback', 'feedback-issue', 'bug'],
```

Comment:
```suggestion
  labels = ['bug'],
```

Last minor thing! This is the default value, which will kick in if the `label` prop is not provided when using the `CreateIssueButton` on a site. Since `feedback` and `feedback-issue` are docs specific, let's default this to `bug` for all other sites. We'll override this default on the docs site when this change is merged in. Other than this, the change looks great!